inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/convolution_gpu_bfyx_f16.cl

```diff
@@ -405,17 +431,31 @@ KERNEL(convolution_bfyx_f16)(
             res = dst;
 #endif
             // TODO Generalize for other block sizes
-#if OUTPUT_X_BLOCK_SIZE == 8
+#if OUTPUT_FORMAT_BFYX
+    #if OUTPUT_X_BLOCK_SIZE == 8
+            vstore8(TO_OUTPUTVTYPE8(res), 0, output + output_offset);
+    #elif OUTPUT_X_BLOCK_SIZE == 4
+            vstore4(TO_OUTPUTVTYPE4(res), 0, output + output_offset);
+    #elif OUTPUT_X_BLOCK_SIZE == 2
+            vstore2(TO_OUTPUTVTYPE2(res), 0, output + output_offset);
```

Comment:
Lets reduce these with a common definition of VSTORE(N)

Reply:
Will fix